ansible/inventories/devnet-11/host_vars/prysm-besu-1-arm.yaml

```diff
@@ -0,0 +1,3 @@
+prysm_container_image: gcr.io/prylabs-dev/prysm/beacon-chain:devnet-10
```

Comment:
still need this?

Reply:
yes. We still wont have an arm prysm img. And we gonna be running arm stuff